samples/subsys/mgmt/mcumgr/smp_svr/README.rst

```diff
@@ -263,6 +263,39 @@ The general sequence of a DFU process is as follows:
 * Reset the device remotely using :file:`mcumgr`
 * Confirm the uploaded image using :file:`mcumgr` (optional)
 
+Slot to image mapping and selection
+===================================
+
+Currently the mcumgr supports 4 images of which first two are mapped into
+primary and secondary slot. Default upload is performed to "image-1", which
+is slot-0, and the application is always running from "image-0", that is slot-1.
```

Comment:
in table below image-1 is same row as slot-1, similar image-0 and slot-0
I don't get what this sentence mean.

Reply:
Ha, because it is wrong.